src/config.h

```diff
@@ -200,7 +205,7 @@ void RSConfig_AddToInfo(RedisModuleInfoCtx *ctx);
     .minUnionIterHeap = 20, .numericCompress = false, .numericTreeMaxDepthRange = 0,              \
     .printProfileClock = 1, .invertedIndexRawDocidEncoding = false,                               \
     .forkGCCleanNumericEmptyNodes = true, .freeResourcesThread = true, .defaultDialectVersion = 1,\
-    .vssMaxResize = 0, .multiTextOffsetDelta = 100,                                               \
+    .vssMaxResize = 0, .multiTextOffsetDelta = 100, .dialects = 0                                 \
```

Comment:
isn't the default dialect `1`?

Reply:
This is a bitfield of used dialects. The default dialect doesn't matter, that's stored in `.defaultDialectVersion`. This is for dialects that have been used in actual queries. Upon instantiation, no queries have yet been performed, so none of the dialect bits are set.